be/src/vec/exprs/lambda_function/varray_map_function.cpp

```diff
@@ -77,9 +77,8 @@ class ArrayMapFunction : public LambdaFunction {
 
     std::string get_name() const override { return name; }
 
-    doris::Status execute(VExprContext* context, doris::vectorized::Block* block,
-                          int* result_column_id, const DataTypePtr& result_type,
-                          const VExprSPtrs& children) override {
+    Status execute(VExprContext* context, vectorized::Block* block, int* result_column_id,
+                   const DataTypePtr& result_type, const VExprSPtrs& children) override {
         LambdaArgs args;
         // collect used slot ref in lambda function body
         _collect_slot_ref_column_id(children[0], args);
```

Comment:
refactor all private func of ArrayMapFunction, only pass the use param, not args